app/scripts/modules/core/src/pipeline/config/CreatePipeline.tsx

```diff
@@ -23,9 +24,11 @@ export class CreatePipeline extends React.Component<ICreatePipelineProps> {
   public render() {
     const { application } = this.props;
 
-    const pipelineConfigs = get(application, 'pipelineConfigs.data', []).filter(
-      (pipelineConfig: IPipeline) => !PipelineTemplateV2Service.isV2PipelineConfig(pipelineConfig),
-    );
+    const pipelineConfigs = SETTINGS.feature.managedPipelineTemplatesV2UI
+      ? get(application, 'pipelineConfigs.data', [])
+      : get(application, 'pipelineConfigs.data', []).filter(
+          (pipelineConfig: IPipeline) => !PipelineTemplateV2Service.isV2PipelineConfig(pipelineConfig),
```

Comment:
maybe replace the ternary with moving the `SETTINGS` check into the filter so that if one were to have MPTv2 enabled at some point but then later disable it, the MPTv2 configs wouldn't show up?

Reply:
I was thinking that toggling the flag on and off would cause the next component render to check for the flag and if true just return all configs or if not return configs with mptv2 ones filtered out. Is that off though, does it work differently in spinnaker?

Reply:
ah sorry i'm being dumb, all good :+1: